### PR TITLE
ipsec: T7562: Add support for `disable-uniqreqids` option in IPsec configs

### PR DESCRIPTION
**Pull request by alexandr-san4ez**
## Change summary
This PR makes `set vpn ipsec disable-uniqreqids` work with the modern StrongSwan backend by setting `unique=never` in **swanctl.conf** for connections. This restores legacy behavior about multiple connections with the same identity.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)
* https://vyos.dev/T7562

## How to test / Smoketest result
```
root@c6392b5716ee:/vyos/vyos-build# make test MATCH="ipsec"

DEBUG - Running Testcase: /usr/libexec/vyos/tests/smoke/cli/test_vpn_ipsec.py
DEBUG - test_dhcp_fail_handling (__main__.TestVPNIPsec.test_dhcp_fail_handling) ... ok
DEBUG - test_dmvpn (__main__.TestVPNIPsec.test_dmvpn) ... ok
DEBUG - test_flex_vpn_vips (__main__.TestVPNIPsec.test_flex_vpn_vips) ... ok
DEBUG - test_remote_access (__main__.TestVPNIPsec.test_remote_access) ... ok
DEBUG - test_remote_access_dhcp_fail_handling (__main__.TestVPNIPsec.test_remote_access_dhcp_fail_handling) ... ok
DEBUG - test_remote_access_eap_tls (__main__.TestVPNIPsec.test_remote_access_eap_tls) ... ok
DEBUG - test_remote_access_no_rekey (__main__.TestVPNIPsec.test_remote_access_no_rekey) ... ok
DEBUG - test_remote_access_pool_range (__main__.TestVPNIPsec.test_remote_access_pool_range) ... ok
DEBUG - test_remote_access_vti (__main__.TestVPNIPsec.test_remote_access_vti) ... ok
DEBUG - test_remote_access_x509 (__main__.TestVPNIPsec.test_remote_access_x509) ... ok
DEBUG - test_retransmission_default_settings (__main__.TestVPNIPsec.test_retransmission_default_settings) ... ok
DEBUG - test_retransmission_settings (__main__.TestVPNIPsec.test_retransmission_settings) ... ok
DEBUG - test_site_to_site (__main__.TestVPNIPsec.test_site_to_site) ... ok
DEBUG - test_site_to_site_ts_protocol_all (__main__.TestVPNIPsec.test_site_to_site_ts_protocol_all)
DEBUG - Test acceptance of 'all' protocol in site-to-site traffic selector. ... ok
DEBUG - test_site_to_site_vti (__main__.TestVPNIPsec.test_site_to_site_vti) ... ok
DEBUG - test_site_to_site_vti_ts_afi (__main__.TestVPNIPsec.test_site_to_site_vti_ts_afi) ... ok
DEBUG - test_site_to_site_with_default_ts (__main__.TestVPNIPsec.test_site_to_site_with_default_ts)
DEBUG - Test 'site to site' with default value of local and remote Traffic Selection ... ok
DEBUG - test_site_to_site_x509 (__main__.TestVPNIPsec.test_site_to_site_x509) ... ok
```

```
root@c6392b5716ee:/vyos/vyos-build# make test MATCH="l2tp"

DEBUG - Running Testcase: /usr/libexec/vyos/tests/smoke/cli/test_vpn_l2tp.py
DEBUG - test_accel_ipv4_pool (__main__.TestVPNL2TPServer.test_accel_ipv4_pool) ... ok
DEBUG - test_accel_ipv6_pool (__main__.TestVPNL2TPServer.test_accel_ipv6_pool) ... ok
DEBUG - test_accel_limits (__main__.TestVPNL2TPServer.test_accel_limits) ... ok
DEBUG - test_accel_local_authentication (__main__.TestVPNL2TPServer.test_accel_local_authentication) ... ok
DEBUG - test_accel_log_level (__main__.TestVPNL2TPServer.test_accel_log_level) ... ok
DEBUG - test_accel_name_servers (__main__.TestVPNL2TPServer.test_accel_name_servers) ... ok
DEBUG - test_accel_next_pool (__main__.TestVPNL2TPServer.test_accel_next_pool) ... ok
DEBUG - test_accel_ppp_options (__main__.TestVPNL2TPServer.test_accel_ppp_options) ... ok
DEBUG - test_accel_radius_authentication (__main__.TestVPNL2TPServer.test_accel_radius_authentication) ... ok
DEBUG - test_accel_shaper (__main__.TestVPNL2TPServer.test_accel_shaper) ... ok
DEBUG - test_accel_snmp (__main__.TestVPNL2TPServer.test_accel_snmp) ... ok
DEBUG - test_accel_wins_server (__main__.TestVPNL2TPServer.test_accel_wins_server) ... ok
DEBUG - test_l2tp_radius_server (__main__.TestVPNL2TPServer.test_l2tp_radius_server) ... ok
DEBUG - test_l2tp_server_authentication_protocols (__main__.TestVPNL2TPServer.test_l2tp_server_authentication_protocols) ... ok
DEBUG - test_vpn_l2tp_dependence_ipsec_swanctl (__main__.TestVPNL2TPServer.test_vpn_l2tp_dependence_ipsec_swanctl) ... ok
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: data/templates/ipsec/swanctl/l2tp.j2 [hunk 1]
```diff
@@ -8,6 +8,9 @@
         dpd_timeout = 45s
         rekey_time = {{ l2tp_ike.lifetime if l2tp_ike else l2tp.ike_lifetime }}s
         reauth_time = 0
+{% if uniqreqids %}
```
Comment (c-po) [inline data/templates/ipsec/swanctl/l2tp.j2]: Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?

File: data/templates/ipsec/swanctl/peer.j2 [hunk 2]
```diff
@@ -32,6 +32,9 @@
 {% endif %}
 {% if peer_conf.force_udp_encapsulation is vyos_defined %}
         encap = yes
+{% endif %}
+{% if uniqreqids %}
```
Comment (c-po) [inline data/templates/ipsec/swanctl/peer.j2]: Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?

File: data/templates/ipsec/swanctl/profile.j2 [hunk 3]
```diff
@@ -13,6 +13,9 @@
         dpd_timeout = {{ ike.dead_peer_detection.timeout }}
         dpd_delay = {{ ike.dead_peer_detection.interval }}
 {%         endif %}
+{%         if uniqreqids %}
```
Comment (c-po) [inline data/templates/ipsec/swanctl/profile.j2]: Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?

**Review by c-po — DISMISSED** [review]

File: data/templates/ipsec/swanctl/l2tp.j2 (hunk 1 above)
Comment (alexandr-san4ez) [reply data/templates/ipsec/swanctl/l2tp.j2]: Yes, your are right. I have updated all of this conditions.

**Review by sever-sever — APPROVED** [review]
Add missing tepmplate and configuration for the `set vpn ipsec disable-uniqreqids` option

**Review by dmbaturin — APPROVED** [review]
Issues are addressed now, no objections.
